client/src/components/Modal/Modal.jsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+
+class Modal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      step: 0,
+      date: '',
+    };
+    this.renderConfirmation = this.renderConfirmation.bind(this);
+    this.renderReview = this.renderReview.bind(this);
+  }
+
+  renderConfirmation(e) {
+    // e.preventDefault();
+    this.setState({
+      step: 1,
+    });
+  }
+
+  renderReview(e) {
+    this.setState({
+      step: 2
+    });
+  }
+
+  render() {
+    const { restaurant } = this.props;
+    const { step } = this.state;
+    if (step === 1) {
+      return (
+
+        <div className="container-confirm">
+          <div className="modal">
+            <button className="close-modal" href="#" onClick={this.renderReview} type="button">X</button>
+            <div className="confirm-box">
+
+              <img className="box-item svg-item" src="https://s3-media2.fl.yelpcdn.com/assets/srv0/reservation_pages/9594f2f4936f/assets/img/www_ResIllustration@2x.png" alt="" />
+              <p className="noteforuser box-item">Hammer, your reservation has been confirmed! </p>
+
+            </div>
+          </div>
+        </div>
+      );
+    }
+    if (step === 0) {
+      return (
```

Comment:
This section is really long. Can you think of ways to break this into smaller components?